### ISSUE TITLE
Strange mapping of moon phases (percentage illuminated to moon phases)

### ISSUE THREAD

**Issue by sjvudp** [forrestguice/SuntimesWidget#385]
Yesterday there was a bright sky and the moon was clearly visible. While wondering whther it's full moon, I started the app to look it up.
To my surprise the textual output was "decreasing three quarter moon" (translated back from German "Abnehmender Dreiviertelmond"). Somehow it did not match the impression. Then today I discovered that the illumination is still displayed as 94.5%, which is definitely more than 75% (three quarters).

So 95,4% would be "decreasing (waning) full moon", and for <75% it would be "decreasing three quarter moon".

For me the pahses should be like (excuse my poor English):

- Illumination 0%: New moon
- Illumination >0% and illumination < 25%: increasing quarter moon
- Illumination 25%: Quarter moon
- Illumination >25 and illumination < 50%: Increasing quarter moon
- Illumination 50% Half moon
- Illumination >50% and illumination < 75%: Increasing half moon
- Illumination 75%: Three-quarter moon
- Illumination >75% and illumination < 100%: Increasing three-quarter moon
- Illumination 100%: Full moon

You could add ±1% tolerance for the "exact" quarter-phases...
![Suntimes_20200112-165740](https://user-images.githubusercontent.com/6734808/72222372-493da580-3564-11ea-9a0c-bf3d87ddf180.jpg)

**Comment by forrestguice** [comment]
Its possible the German translation is incorrect. Its been reviewed a few times though.

The illumination table you've listed is what is expected (but the labels you are using aren't very familiar to me). In English it is.. Full Moon -> Waning Gibbous -> Third Quarter -> Waning Crescent -> New Moon -> Waxing Crescent -> First Quarter -> Waxing Quarter. https://www.timeanddate.com/calendar/aboutmoonphases.html

**Comment by forrestguice** [comment]
Actually that table looks strange to me. This is what is expected:
    Illumination 0%: New moon
    Illumination >0% and illumination < 50%: waxing crescent moon
    Illumination 50%: Quarter moon
    Illumination >50 and illumination < 100%: waxing gibbous moon
    Illumination 100% Full Moon
    Illumination >100% and illumination < 50%: Waning gibbous moon
    Illumination 50%: Three-quarter moon
    Illumination >50% and illumination < 0%: Waning Crescent Moon
    Illumination 0%: New moon

I'm not sure what a "half moon" is since that is just "first quarter" or "third quarter" "(at which the illumination is actually 50%). I think maybe something is "lost in translation" or there are some misconceptions of the phases.

**Comment by sjvudp** [comment]
> but the labels you are using aren't very familiar to me

Agreed, the English variants seem less regular than in German. It's somewhat a matter of taste whether the smallest increment to consider is a quarter (or even less) or a half.
(one eight of the moon is sometimes named "Sichelmond" (sickle moon))
In German you just have "zunehmend.." (increasing) and "abnehmend.." (decreasing).

So you could simply divide into increasing moon (0% to 100%) when "the light is on the right" and decreasing moon (100% to 0 %) when "the light is on the left" in German.
The additional labels are somewhat culturally defined; in German like using as a reference, so "increasing quarter moon" would be "more than quarter moon" (25% + x), up to "half moon" ("half full", or "half empty", as you like).

Maybe part of the confusion is that the German cycle seems to be based on illumination (where full moon is 100%), while the cycle at https://www.timeanddate.com/calendar/aboutmoonphases.html considers full moon to be half of the cycle.

So "first quarter" would be "zunehmender Halbmond" (increasing half moon, 50% + x) IMHO, and "third quarter" would be "abnehmender Halbmond" (decreasing half moon, 50% - x) IMHO.

Having had a look at https://github.com/forrestguice/SuntimesWidget/blob/master/app/src/main/res/values-de/strings.xml, I also think the translations are OK, but going back to the illumination reported being roughly 94%, that would be between "Full Moon" and "Waning Gibbous Moon", but more closer to "Full Moon". So IMHO it would be "abnehmender Vollmond" ("decreasing full moon", 100% - x).
